__tests__/app.test.js

```diff
@@ -502,3 +502,41 @@ describe(`PATCH /api`, () => {
         });
     });
 });
+
+describe(`DELETE /api`, () => {
+    describe(`/comments/:comment_id`, () => {
+        test('returns status 204 with no content and deletes the given comment', () => {
+            return request(app)
+                .delete('/api/comments/1')
+                .expect(204)
+                .then(() => {
+                    return request(app)
+                        .get('/api/comments')
+                        .expect(200)
+                        .then(({ body }) => {
+                            const { comments } = body;
+                            expect(comments.length).toBe(5);
+                        });
+                });
+        });
```

Comment:
It's smart that you've considered how to check this here and make sure that the comment has been removed. Just consider how quick this will be if we're making multiple requests to the db. This is a decision that you need to make as a dev, but I'd always try and make as few requests as possible.